src/payment/strategies/moneris/moneris.ts

```diff
@@ -1,9 +1,43 @@
+/**
+ * A set of stringified CSS to apply to Moneris' IFrame fields.
+ * CSS attributes should be converted to string.
+ * Please note that ClassNames are not supported.
+ *
+ * IE:
+ * ```js
+ * {
+ *      cssBody: 'background:white;';
+ *      cssTextbox: 'border-width:2px;';
+ *      cssTextboxPan: 'width:140px;';
+ *      cssTextboxExpiry: 'width:40px;';
+ *      csstexboxCvd: 'width:40px;';
+ * }
+ * ```
+ *
+ * When using several attributes use semicolon to separate each one.
+ * IE: 'background:white;width:40px;'
+ */
 export default interface MonerisStylingProps {
+    /**
+     * Stringified CSS to apply to the body of the IFrame.
+     */
     cssBody?: string;
+    /**
+     * Stringified CSS to apply to each of input fields.
+     */
     cssTextbox?: string;
+    /**
+     * Stringified CSS to apply to the card's number field.
```

Comment:
Why do we use both `Card Number` and `Pan` for the same thing?

Reply:
Card number is the BC wording, Pan is the word that Moneris use for that